### PR TITLE
Update to using a packed Stdext

### PR DESCRIPTION
**Pull request by jonludlam**
Signed-off-by: Jon Ludlam <jonathan.ludlam@citrix.com>

### CODE REVIEW

File: opam [hunk 1]
```diff
@@ -12,7 +19,7 @@ depends: [
   "rpc"
   "fd-send-recv"
   "uuidm"
-  "xapi-stdext"
+  "xapi-stdext" {>= "2.0.0"}
```
Comment (simonjbeaumont) [inline opam]: +1

**Comment by simonjbeaumont** [comment]
Looks good. Guess we'll be needing a spec file update too...

**Comment by jonludlam** [comment]
CI will fail as the packed stdext isn't in the yum repo yet. Chicken and egg and whatnot...

**Comment by simonjbeaumont** [comment]
:chicken: :egg:

**Comment by jonludlam** [comment]
Uh oh, don't merge this!

**Pull request merged**